test_conformance/computeinfo/main.c

```diff
@@ -670,15 +677,20 @@ int getConfigInfos( cl_device_id device )
           err = parseVersion( info.config.string, & version );
           if ( err ) {
             total_errors++;
+            free(info.config.string);
             break;
           }
         } else if ( info.opcode == CL_DEVICE_EXTENSIONS ) {
           err = parseExtensions( info.config.string, & extensions );
           if ( err ) {
             total_errors++;
+            free(info.config.string);
             break;
           }
         }
+        if (info.config_type == type_string) {
```

Comment:
What about spaces around `(` and `)` to conform with code style in surrounding code?